src/extension.js

```diff
@@ -256,6 +277,17 @@ async function connectToSSH()
 		});
 }
 
+function readCTags(conf_ctagssh) {
+	if (CTagSSH_Tags === undefined) {
```

Comment:
If we move reading tags to separate function then we will clear CTagsSSH_Tags before load new tags

Reply:
done